guides/source/action_view_overview.md

```diff
@@ -219,7 +203,11 @@ One way to use partials is to treat them as the equivalent of subroutines; a way
 <%= render "application/footer" %>
 ```
 
-Here, the `_ad_banner.html.erb` and `_footer.html.erb` partials could contain content that is shared among many pages in your application. You don't need to see the details of these sections when you're concentrating on a particular page.
+Here, the `_ad_banner.html.erb` and `_footer.html.erb` partials could contain content that is shared among many pages in your application. You don't need to see the details of these sections when you're focused on a Products' page.
+
+The above example also uses the `_product.html.erb` partial. This partial contains details for rendering an individual product and is used to render each product in the collection `@products`.
+
+### Partial Template Inheritance
```

Comment:
Yup. Yeah this has been a bit of back and forth for me, about all the "Partial
" related sections. But this _specific_ section "Partial Template Inheritance" is clear that it does *not* belong up here, in this location in the guide. It can just be in Layouts and Rendering and I may mention a little something about "template inheritance" at the *bottom* of this guide. But nothing about inheritance needs to up here.